course/models.py

```diff
@@ -102,12 +113,18 @@ def get_user_status(user):
 
 
 class UserStatus(models.Model):
-    user = models.OneToOneField(User, db_index=True, related_name="user_status")
+    user = models.OneToOneField(User, db_index=True, related_name="user_status",
+            verbose_name = 'User ID')
     status = models.CharField(max_length=50,
-            choices=USER_STATUS_CHOICES)
+            choices=USER_STATUS_CHOICES,
+            verbose_name = 'User status')
     sign_in_key = models.CharField(max_length=50,
-            null=True, unique=True, db_index=True, blank=True)
-    key_time = models.DateTimeField(default=now)
+            help_text="NEED HELP TEXT",
+            null=True, unique=True, db_index=True, blank=True,
+            verbose_name = 'Sign in key')
+    key_time = models.DateTimeField(default=now,
+            help_text="NEED HELP TEXT",
+            verbose_name = 'Key time')
 
```

Comment:
Actually, I see this in admin, and I will try to add help text if you don't remove them from admin.